fedimint-client/src/module/mod.rs

```diff
@@ -959,5 +960,47 @@ impl AsRef<maybe_add_send_sync!(dyn IClientModule + 'static)> for DynClientModul
     }
 }
 
+/// A contiguous range of input/output indexes
+#[derive(Copy, Clone, Encodable, Decodable, PartialEq, Eq, Hash, Debug)]
+pub struct IdxRange {
+    start: u64,
+    end_inclusive: u64,
+}
```

Comment:
Would it make sense to have the txid in here too? That way you could decide to iterate over it as `Iter<Item=OutPoint>` or `Iter<Item=u64>`. Afaik all places where we use the range now also have a `Txid`.

Reply:
Can try `OutPointRange` type.

Reply:
@elsirion Done.